api/client.go

```diff
@@ -42,6 +42,7 @@ const (
 	EnvVaultToken         = "VAULT_TOKEN"
 	EnvVaultMFA           = "VAULT_MFA"
 	EnvRateLimit          = "VAULT_RATE_LIMIT"
+	EnvHTTPSProxy         = "VAULT_HTTPS_PROXY"
```

Comment:
Since we don't enforce that this will be using scheme `https`, it feels like the more general name for the var would be VAULT_HTTP_PROXY.